client/src/global/colors.ts

```diff
@@ -0,0 +1,9 @@
+const colors = {
+  urgency: {
+    low: '#5cd060',
+    medium: '#d18f00',
+    high: '#e01300',
```

Comment:
Can you use values for those colors from variables.scss?

Reply:
It's not possible, as far as I know.

Reply:
That's why I basically redeclared them here